### PR TITLE
Refactor/css consolidation

### PR DESCRIPTION
**Pull request by michelecomfort**
What (if any) features are you implementing?
- Added key props to child components
What (if anything) did you refactor?
- Removed repetitive css styles by grouping elements together
- Changed NavLink to Link for navigation purposes
- 
Were there any issues that arose?
-
Is there anything that you need from your teammate?
-
Any other comments, questions, or concerns?
-

### CODE REVIEW

**Review by raz-joan — APPROVED** [review]
Wow. @michelecomfort  🙌🏻 Super thorough clean up of our code base. Thanks for this, especially the CSS!

File: src/Components/Drink.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ const Drink = () => {
     })
 
     return (
-        <div aria-label="Random Drink Result" className="paired-component">
+        <div aria-label="Random Drink Result" className="paired-component" key={currentDrink.idDrink}>
```
Comment (colganmeanor) [inline src/Components/Drink.js:59]: be gone console errors!

**Review by colganmeanor — COMMENTED** [review]
Great work Michele - thank you for being our CSS Queen!

**Review by colganmeanor — APPROVED** [review]

**Comment by colganmeanor** [comment]
Moige Confoimed!

**Pull request merged**